pytato/array.py

```diff
@@ -1678,6 +1736,21 @@ def dtype(self) -> np.dtype[Any]:
 
 # {{{ end-user facing
 
+def _get_default_axes(ndim: int) -> AxesT:
+    return tuple(Axis(_) for _ in range(ndim))
```

Comment:
```suggestion
    return tuple(Axis(i) for i in range(ndim))
```
`_` is convention for "unused", so I find it strange to see an unused thing used.

Reply:
Oops. that was supposed to be `Axis(frozenset())`. But now, I'm confused if we should make the axis dimension an attribute of the `Axis` class. Wdyt?

Reply:
Ok, thanks. Leaving as is.